src/util/address.rs

```diff
@@ -119,7 +122,8 @@ impl std::error::Error for Error {
             | InvalidWitnessProgramLength(_)
             | InvalidSegwitV0ProgramLength(_)
             | UncompressedPubkey
-            | ExcessiveScriptSize => None,
+            | ExcessiveScriptSize 
```

Comment:
This line has trailing whitespace. You should be able to configure what ever system you use to view diffs before pushing them (editor or shell) to highlight trailing white space. In case its useful, I personally view diffs using
```bash
git-log-patch () {
    local commit
    if (( $# < 1 ))
    then
        commit="HEAD~"
    else
        commit="$1"
    fi
    git log --color=always --patch --reverse "${commit}".. | vimpager
}
```

Reply:
Along with a shell alias :)

Reply:
@tcharding Hey, this is good. Thanks a lot! Though I modified it to (`~/.gitconfig`):

```
[alias]
check = "!f() { git log --color=always --patch --reverse "${1:-HEAD~}"..; }; f"
```

That pager seems unnecessary BTW, `git log` pages by default(?) when run on interactive shell.

Reply:
Oh nice, I started using `vimpager` because I was loosing colours somewhere but seems you are correct. I can stop using it, cheers.

Reply:
Thanks for this snippet @dpc and @tcharding ! TIL.